### PR TITLE
post: Is there a course on X

### PR DESCRIPTION
**Pull request by cappe987**
The idea for this post came from a recent discussion in the Rust channel in the Discord server. I reused a cover image from another post, I hope that's ok. I can remove the joke about Boots if you want.

As I started making this PR I read the contributing guidelines and realized you don't want a wall of text, but I'll submit this now and if you don't like it I can try to rewrite it.

### CODE REVIEW

File: content/education/is-there-a-course-on.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Even universities with their 3-4 year degrees only cover a fraction of all
+there is. Instead, it prepares you to specialize and learn what you want.
+
+## Learn on your own
+
+Boot.dev has a similar philosophy as universities in regards to learning. When
+you have completed the Boot.dev curriculum you should be capable to continue
+learning on your own. You'll have a solid foundation to stand on and the skills
+to continue growing them. The curriculum didn't teach you about framework X or
+language Y, but it also doesn't need to. At this point, nothing is stopping you
```
Comment (skovranek) [inline content/education/is-there-a-course-on.md:24]: tiny nit: How about a little bit more emphasis on the futility of chasing shifting trends?

The curriculum didn't teach you about currently trending framework X or language Y, but it also doesn't need to.

**Review by skovranek — CHANGES_REQUESTED** [review]
Nice and to the point. Left a few nit comments.

Please reformat it without line breaks.

**Comment by cappe987** [comment]
I just now saw you had removed your comments haha. I reformatted without line breaks so each paragraph is on a single line to match with the rest of the codebase. This doesn't matter according to the CommonMark spec (and most other implementations) used by Hugo. A single line break formats it the same as no line break. https://spec.commonmark.org/0.17/#paragraphs

I like to wrap my lines since that makes the Vim movement with lines correct. A line only visually wrapped in the editor looks like multiple lines but Vim treats it as one.

**Comment by cappe987** [comment]
Also updated the date of the post to today.

**Pull request merged**